app/lib/features/comments/models.dart

```diff
@@ -0,0 +1,5 @@
+enum NewCommentLocation {
+  hidden,
+  before,
+  after,
+}
```

Comment:
Checked use-case of this `emun` in below files. Look bit weird to me. Let me think what can be done here.